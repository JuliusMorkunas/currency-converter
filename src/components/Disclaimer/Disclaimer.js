import React from 'react';
import s from './Disclaimer.module.scss';

const Disclaimer = ({ children }) => {
  return <p className={s.Disclaimer}>{children}</p>;
};

export default Disclaimer;
