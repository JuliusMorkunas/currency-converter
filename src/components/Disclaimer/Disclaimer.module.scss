@import '../../variables';

.Disclaimer {
  font-size: 12px;
  line-height: 16px;
  margin-top: 0;
  margin-bottom: 4px;
  color: $c-grey-darken;
}
