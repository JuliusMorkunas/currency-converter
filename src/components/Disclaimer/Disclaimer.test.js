import React from 'react';
import { render } from '@testing-library/react';
import Disclaimer from './Disclaimer';

it('Disclaimer should match snapshot', () => {
  const { asFragment } = render(<Disclaimer />);
  expect(asFragment()).toMatchSnapshot();
});
