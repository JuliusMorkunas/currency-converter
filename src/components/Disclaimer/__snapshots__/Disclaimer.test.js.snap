// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disclaimer should match snapshot 1`] = `
<DocumentFragment>
  <p
    class="Disclaimer"
  />
</DocumentFragment>
`;
